config.py

```diff
@@ -20,3 +20,5 @@
 SCHEMA_VERSION = '4.0.0'
 SCHEMA_BASE_URI = 'https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/'
 
+ONTOLOGY_SCHEMA_BASE_URL = 'https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/v5_prototype/json_schema/common/ontology'
```

Comment:
None of the example data is going to validate against v5_prototype as we're going to use v4.0.0 (or possibly a patch release thereof) in the demo. Please point the validator at the latest release for now.

Reply:
Ok sounds good. The code depends on the patch to v4.0.0; changing the ontology classes to use the purl curies